### DISCUSSION TITLE
Connection to Webdav (Https)

### DISCUSSION THREAD

**Discussion by jonhnyyip** [iterate-ch/cyberduck#15763] [Q&A]
Hi, 
I'm trying to connect to my Webdav (https) using mountainduck. But it alway prompt for username and password.
I'm using Window 10 with smartcard login. Is there any other method to map my Webdav drive?

**Comment by dkocher** [comment]
Not sure if your use case is to have a client certificate on your smart card that you want to use to authenticate with the WebDAV Server using mutual TLS. Can you select the certificate you want to use for client authentication from the dropdown list in _Client Certificate_ when editing the bookmark?
